foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -587,10 +589,10 @@ msgid ""
 "all your devices. It usually has less interference, but the signal does not "
 "carry so well indoors."
 msgstr ""
-"Das 2,4 GHz Frequenzband ist unter den Clientgeräten breiter unterstützt, es "
-"ist aber mehr durch Interferenzstörungen belastet. Das 5 GHz Frequenzband "
-"ist durch eine neuere Norm definiert, welche nicht von aller Ihrer Geräte "
-"unterstützt sein mag. Gewöhnlich ist dieses Frequenzband weniger gestört, es "
+"Das 2,4 GHz Frequenzband ist unter den Clientgeräten besser unterstützt, es "
+"ist aber mehr durch Interferenzen belastet. Das 5 GHz Frequenzband ist "
+"neuerer Standard, welche aber eventuell nicht von allen Ihren Geräte "
```

Comment:
Shouldn't there be an indefinite article ('ist ein neuerer Standard') then? IMO, 5 GHz is not an standard - it's a frequency range.

Also allen Ihren Geräten. (new form is missing the 'n')